innopoints/config/prod.py

```diff
@@ -1,13 +1,14 @@
-"""Application configuration for production"""
+"""Application configuration for production."""
 
 import os
 from innopoints.config.common import *
 
+
 INNOPOLIS_SSO_CLIENT_ID = os.environ['INNOPOLIS_SSO_CLIENT_ID']
 INNOPOLIS_SSO_CLIENT_SECRET = os.environ['INNOPOLIS_SSO_CLIENT_SECRET']
 
-def is_admin(userinfo):
-    """Determine if the user is an administrator by a dictionary of claims"""
+def is_admin(userinfo):  # pylint: disable=unused-argument
+    """Determine if the user is an administrator by a dictionary of claims."""
     return False
 
 IS_ADMIN = is_admin
```

Comment:
Make it consistent with dev.py, where you use a lambda function instead.
Also, shouldn't it read a list of admin emails from the environment variable?

Reply:
That's the reason for the inconsistency. The production config is simply not ready, but it will check the group. Keep it like that for now.